_episodes/59.md

```diff
@@ -0,0 +1,169 @@
+tags: trino
+youtube_id: "UH9adsSyV4Q"
+wistia_id: "kv548c9fnd"
+sections:
+- title: "Introduction and Manfred's Seattle trip"
+  time: 0:00
+- title: Trino 445
+  time: 1:46
+- title: Our guest - Lukas Eder
+  desc: Java expert, Java Champion and creator of jOOQ
+  time: 3:26
+- title: High-level intro to jOOQ
+  time: 7:32
+- title: data geekery history and more jOOQ architecture
+  time: 9:32
+- title: Other similar systems
+  desc: Java ORMs, object graph persistence, and comparison with Hibernate
+  time: 11:51
+- title: jOOQ for existing databases
+  time: 13:04
+- title: Query rewriting
+  time: 14:59
+- title: Other project from data geekery
+  time: 16:47
+- title: jOOQ community, users, customers, and more
+  time: 19:05
+- title: More comparisons
+  desc: Hibernate and MyBatis
+  time: 21:27
+- title: Download, distributions, features
+  time: 23:42
+- title: querydsl
+  time: 26:05
+- title: Support for different database
+  desc: And discussion around SQL dialects
+  time: 27:14
+- title: Trino details
+  desc: Mmory connector, data types, type safety, and other aspects
+  time: 28:49
+- title: Audience comment
+  desc: Discussion about non-relational features moving to RDBMS
+  time: 33:07
+- title: Audience comment
+  desc: User question about jOOQ and Trino support
+  time: 35:20
+- title: Trino connectors,  jOOQ, testing, and issues
+  time: 37:15
+- title: Different databases and function support
+  time: 40:42
+- title: Demo
+  desc: Including the fluent API, type safety, pojos, daos, ...
+  time: 43:50
+- title: Rounding out with announcements
+  time: 1:01:18
+- title: Trino Fest sponsors, speakers, and talks
+  time: 1:02:16
+---
+
+## Host
+
+* [Manfred Moser](https://www.linkedin.com/in/manfredmoser), Director of Trino
+  Community Leadership at [Starburst](https://starburst.io),
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guest
+
+* Lukas Eder, Creator of [jOOQ](https:/jooq.org),
+  ([@lukaseder](https://twitter.com/lukaseder))
+
+## Trino releases
+
+[Trino 445](https://trino.io/docs/current/release/release-445.html)
+
+* Add support for time travel queries with the Delta Lake connector.
+* Add support for the `REPLACE` modifier as part of a `CREATE TABLE` statement
+  with the Delta Lake connector.
+* Add support for writing Bloom filters in Parquet files with the Hive connector.
+* Add support for dynamic filtering to the MongoDB connector.
+* Expand support for function pushdown in the Snowflake connector.
+
+## Lukas Eder and data geekery
+
+[Lukas](https://twitter.com/lukaseder) is recognized as a Java Champion and
+well-known as a very active member of the Java community. We chat about his
+history and involvement in the community of Java and related open source
+projects, and how it lead to [jOOQ and his company data
+geekery](https://www.jooq.org/). Lukas also briefly talks about other products.
+
+## jOOQ
+
+jOOQ stands for jOOQ Object Oriented Querying (jOOQ). It generates Java code
```

Comment:
```suggestion
jOOQ stands for Java Object Oriented Querying (jOOQ). It generates Java code
```
I assume? Unless it really is recursion all the way down

Reply:
Its recursion ...

Reply:
Yes, see also: https://www.jooq.org/how-to-pronounce-jooq